### ISSUE TITLE
Keyboard support for brush

### ISSUE THREAD

**Issue by julianna-ciq** [recharts/recharts#3549]
- [x] I have searched the [issues](https://github.com/recharts/recharts/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
Users should be able to tab to each grip of the brush. Users should be able to use left/right arrow keys to adjust the grips, in the same way mouse users can grab them.
Additionally, screen reader users should be informed that those grips are range sliders. They should know what the current value is, and should be updated as the value changes.

For additional reference, see [W3C's WAI-ARIA "Slider Pattern" authoring guidelines](https://www.w3.org/WAI/ARIA/apg/patterns/slider/).

### What does the proposed API look like?
There should be no change necessary to the API. The slider's elements should be updated to support keyboard navigation and ARIA attributes.

**Comment by ckifer** [comment]
merged in https://github.com/recharts/recharts/pull/3633

**Comment by ckifer** [comment]
released in 2.8
